devsquad-cockpit/laravel
# COC-121: Remove Pest

## What?

This PR removes the Pest package and now uses PHPunit for all tests.

- [JIRA-121](https://team-devsquad.atlassian.net/browse/COC-121)

## Why?

Because some Pest packages only work for PHP 8.0

## Checklist:

#### Required to set up the environment to test this PR


#### Have you done all items below?


## How can it be tested?

- Go to this branch
- Run `composer verify`
